src/System.Private.CoreLib/shared/System/Threading/Tasks/Task.cs

```diff
@@ -6110,6 +6070,9 @@ internal static Task<TResult> CreateUnwrapPromise<TResult>(Task outerTask, bool
             return new UnwrapPromise<TResult>(outerTask, lookForOce);
         }
 
+#if CORERT
```

Comment:
This should be `#if PROJECTN`. These attributes are just for .NET Native. CoreRT does not need them.

Reply:
Done